docs/vision-usage.rst

```diff
@@ -0,0 +1,190 @@
+  :envvar:`GCLOUD_PROJECT` environment variable for the project you'd like
+  to interact with. If you are Google App Engine or Google Compute Engine
+  this will be detected automatically.
+
+- After configuring your environment, create a
+  :class:`Client <gcloud.vision.client.Client>`
+
+  .. doctest::
+
+     >>> from gcloud import vision
+     >>> client = vision.Client()
+
+  or pass in ``credentials`` and ``project`` explicitly
+
+  .. doctest::
+
+     >>> from gcloud import vision
+     >>> client = vision.Client(project='my-project', credentials=creds)
+
+Annotating an Image
+-------------------
+
+Annotate a single image
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. doctest::
+
+    >>> from gcloud import vision
+    >>> client = vision.Client(project="my-project-name")
+    >>> with open('/tmp/car.jpg', 'r') as f:
+    ...     client.annotate(f.read(), vision.FeatureTypes.LABEL_DETECTION, 3)
+
+Annotate multiple images
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. doctest::
+
+     >>> images = (
+     ...     ('./image.jpg', [
+     ...         vision.FeatureTypes.LABEL_DETECTION,
+     ...         vision.FeatureTypes.LANDMARK_DETECTION]),
+     ...     ('./image2.jpg', [
+     ...         vision.FeatureTypes.FACE_DETECTION,
+     ...         vision.FeatureTypes.TEXT_DETECTION]),)
+     >>> annotated_images = []
+     >>> for image, feature_types in images:
+     ...     annotated_images.append(
+     ...         vision_client.annotate(
+     ...            image,
+     ...            feature_types))
+
+Failing annotations return no results for the feature type requested.
+
+.. doctest::
+
+    >>> from gcloud import vision
+    >>> client = vision.Client(project="my-project-name")
+    >>> with open('/tmp/car.jpg', 'r') as f:
+    ...     results = client.annotate(f.read(),
+    ...                               vision.FeatureTypes.LOGO_DETECTION, 3)
+    >>> len(results.logos) # 0
+
+Face Detection
+~~~~~~~~~~~~~~
+
+Annotating using the ``FACE_DETECTION`` feature type.
+
+
+.. doctest::
+
+    >>> from gcloud import vision
+    >>> client = vision.Client(project="my-project-name")
+    >>> with open('/tmp/car.jpg', 'r') as f:
+    ...     results = client.annotate(f.read(),
+    ...                              vision.FeatureTypes.FACE_DETECTION, 3)
+    >>> results.faces[0].landmarks[0].type # LEFT_EYE
+    >>> results.faces[0].landmarks[0].position.x # 1301.2404
+    >>> results.faces[0].detection_confidence # 0.9863683
+    >>> results.faces[0].joy_likelihood # 0.54453093
+
+
+Label Detection
+~~~~~~~~~~~~~~~
+
+Annotating using the ``LABEL_DETECTION`` feature type.
+
+.. doctest::
+
+    >>> from gcloud import vision
+    >>> client = vision.Client(project="my-project-name")
+    >>> with open('/tmp/car.jpg', 'r') as f:
+    ...     results = client.annotate(f.read(),
+    ...                              vision.FeatureTypes.LABEL_DETECTION, 3)
+    >>> results.labels[0].description # automobile
```

Comment:
If this is a print, just provide the output (this is supposed to be console output, after all). So instead

```python
    >>> results.labels[0].description
    'automobile'
```